config/groups.yaml

```
@@ -639,3 +639,7 @@
     - "2575138"
     - "2522944"
   id: 190041
+- name: "Group LuNiJo"
+- members:
```

This isn't right: each `-` denotes an item in a list, so you have added two items here, but both `name` and `members` are supposed to be part of the same item in the list. 

```suggestion
  members:
```